crates/uv-resolver/src/requires_python.rs

```diff
@@ -160,6 +160,23 @@ impl RequiresPython {
         &self.bound
     }
 
+    /// Returns the [`RequiresPythonBound`] truncated to the major and minor version.
+    pub fn bound_major_minor(&self) -> RequiresPythonBound {
+        match self.bound.as_ref() {
+            // Ex) `>=3.10.1` -> `>=3.10`
+            Bound::Included(version) => RequiresPythonBound(Bound::Included(Version::new(
+                version.release().iter().take(2),
+            ))),
+            // Ex) `>3.10.1` -> `>=3.10`
+            // This is unintuitive, but `>3.10.1` does indicate that _some_ version of Python 3.10
+            // is supported.
+            Bound::Excluded(version) => RequiresPythonBound(Bound::Included(Version::new(
```

Comment:
This is strange but I think correct...

Reply:
Yeah `>3.10` is a footgun because it does actually include `3.10` version, this is basically PYI006